### PR TITLE
Miscellaneous code inspection fixes

### PR DESCRIPTION
**Pull request by fviernau**
See individual commits.

### CODE REVIEW

File: clients/vulnerable-code/src/main/kotlin/VulnerableCodeService.kt [hunk 1]
```diff
@@ -40,7 +40,7 @@ interface VulnerableCodeService {
         /**
          * The mapper for JSON (de-)serialization used by this service.
          */
-        val JSON_MAPPER = JsonMapper().setPropertyNamingStrategy(PropertyNamingStrategies.SNAKE_CASE)
+        private val JSON_MAPPER = JsonMapper().setPropertyNamingStrategy(PropertyNamingStrategies.SNAKE_CASE)
```
Comment (sschuberth) [inline clients/vulnerable-code/src/main/kotlin/VulnerableCodeService.kt:43]: Please keep this as-is [as already discussed in one of your previous code inspection fix PRs](https://github.com/oss-review-toolkit/ort/pull/3636#discussion_r576092369).

**Review by sschuberth — CHANGES_REQUESTED** [review]

File: clients/vulnerable-code/src/main/kotlin/VulnerableCodeService.kt (hunk 1 above)
Comment (fviernau) [reply clients/vulnerable-code/src/main/kotlin/VulnerableCodeService.kt]: Damn, my brain refuses to remember these kind of edge cases. Done.

**Review by sschuberth — DISMISSED** [review]

**Review by sschuberth — APPROVED** [review]

**Review by sschuberth — APPROVED** [review]

**Comment by fviernau** [comment]
Merge despite the missing statuses of the static analysis. 
This happened repeatedly but the jobs were successful before the rebases.

**Pull request merged**